docs/en/tutorials/customize_datasets.md

```diff
@@ -172,3 +172,39 @@ data = dict(
 )
 
 ```
+
+### Multi-image Mix Dataset
+
+We use `MultiImageMixDataset` as a wrapper to mix images from multiple datasets.
+`MultiImageMixDataset` can be used by multiple images mixed data augmentation
+like mosaic and mixup.
+
+An example of using `MultiImageMixDataset` with `Mosaic` data augmentation:
+```python
```

Comment:
![image](https://user-images.githubusercontent.com/76149310/150634286-e99782d8-e7bb-4b53-8284-451cf27d901a.png)

MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines